tests/eosiod_run_test.py

```diff
@@ -65,16 +68,19 @@ def cmdError(name, code=0, exitNow=False):
 dumpErrorDetails=args.dump_error_details
 keepLogs=args.keep_logs
 dontLaunch=args.dont_launch
+skipInit=args.skip_init
+skipCreateAccount=args.skip_create_account
 dontKill=args.dont_kill
 
 testUtils.Utils.Debug=debug
 localTest=True if server == LOCAL_HOST else False
-cluster=testUtils.Cluster(walletd=True, enableMongo=enableMongo, initaPrvtKey=initaPrvtKey, initbPrvtKey=initbPrvtKey)
-walletMgr=testUtils.WalletMgr(True)
+cluster=testUtils.Cluster(walletd=True, enableMongo=enableMongo, initaPrvtKey=initaPrvtKey, initbPrvtKey=initbPrvtKey, host=server, port=port)
+walletMgr=testUtils.WalletMgr(True, port, server)
 
 testSuccessful=False
+killEosInstances=True
+killWallet=not args.keep_wallet
 killEosInstances=not dontKill
-killWallet=not dontKill
 
```

Comment:
Can you tie killWallet with both keep_wallet and don't-kill so users don't need to set both to leave the cluster standing? That was the original behavior.